app/lib/experiments.coffee

```diff
@@ -79,101 +72,45 @@ module.exports = class ExperimentServerClient
     @excludedReason = null
     @currentCohort = null
     @currentParticipant = null
-    Geordi.UserGetter.currentUserID = null
-
-  ###
-  when we first get participant, and the user has not started experiment in a previous sessions, we'll need to log it to Geordi
-  ###
-  checkForExperimentStartAndLogIt: (participant) ->
-    if participant.blank_subjects_seen.length==0 && participant.non_blank_subjects_seen.length==0
-      Geordi.logEvent 'experimentStart'
+    @Geordi.UserStringGetter.currentUserID = null
 
   ###
-  when we first discover that a user has ended the experiment, we log it to Geordi
+  Check if current user is eligible to participate in experiments
   ###
-  checkForExperimentEndAndLogIt: (participant) ->
```

Comment:
this participant-tracking code was specific to the blanks experiment and is no longer needed.